src/Gui/Workbench.cpp

```diff
@@ -839,7 +849,7 @@ ToolBarItem* StdWorkbench::setupCommandBars() const
     auto view = new ToolBarItem( root );
     view->setCommand("Standard views");
     *view << "Std_ViewFitAll" << "Std_ViewFitSelection" << "Std_ViewIsometric" << "Separator"
-          << "Std_ViewFront" << "Std_ViewRight" << "Std_ViewTop" << "Separator"

```

Comment:
I don't know what are CommandBars. Why are you removing the separator here?
Same as above comment, I was asked to put the separator back. So better not remove it I guess